src/views/dialogs/OnboardingDialog/ChooseWallet.tsx

```diff
@@ -46,22 +47,34 @@ export const ChooseWallet = () => {
         </Styled.AlertMessage>
       )}
 
-      <Styled.Wallets>
-        {displayedWallets.map((walletType) => (
-          <Styled.WalletButton
-            action={ButtonAction.Base}
-            key={walletType}
-            onClick={() => selectWalletType(walletType)}
-            slotLeft={<Styled.Icon iconComponent={wallets[walletType].icon} />}
-            size={ButtonSize.Small}
-          >
-            {walletType === WalletType.Discord && 'Discord'}
-            {walletType === WalletType.Email && 'Email'}
-            {walletType === WalletType.Twitter && 'Twitter'}
-            {![WalletType.Discord, WalletType.Email, WalletType.Twitter].includes(walletType) && stringGetter({ key: wallets[walletType].stringKey })}
-          </Styled.WalletButton>
-        ))}
-      </Styled.Wallets>
+      {selectedWalletType === WalletType.Email ? (
+        <div>email</div>
```

Comment:
This is just a placeholder right?

Reply:
yes

Reply:
i actually added this by accident, i'll rm